icu4c/source/common/unicode/utrace.h

```diff
@@ -113,10 +113,35 @@ typedef enum UTraceFunctionNumber {
 #endif  // U_HIDE_DEPRECATED_API
 
 #ifndef U_HIDE_DRAFT_API
+
+    /**
+     * The lowest resource/data location.
```

Comment:
"location" seems like a misnomer

Reply:
This is pre-existing language.  See the other UTRACE macros.